crates/time/Cargo.toml

```diff
@@ -12,5 +12,9 @@ readme = "README.md"
 rust-version = "1.81.0"
 
 [dependencies]
+quickcheck = "1.0.3"
```

Comment:
We're already using `proptest` in other parts of the codebase; Although I am reasonably fond of Haskell's QuickCheck, I didn't really find the Rust version to be as convenient / practical to use. 

In particular, QuickCheck gets most of its value when values are properly shrunk, though here it seems you've completely skipped shrinking altogether 😅 ... `proptest` is a bit more Rust-idiomatic (although the API is very macro-based), and comes with integrated shrinking. 

Is there any reason to stick with QuickCheck that I miss here? If not, let's switch to proptest (and possibly start some kind of _contribution guidelines_ with these elements ... I am sorry to come late with this).